src/Uno.UI/UI/Xaml/UIElement.Pointers.cs

```diff
@@ -567,7 +755,12 @@ private bool OnPointerUp(PointerRoutedEventArgs args, bool isManagedBubblingEven
 				// We need to process only events that are bubbling natively to this control (i.e. isOverOrCaptured == true),
 				// if they are bubbling in managed it means that they where handled a child control,
 				// so we should not use them for gesture recognition.
+				var isDragging = _gestures.Value.IsDragging;
```

Comment:
Codacy found an issue: [Remove the unused local variable 'isDragging'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6270104)